torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -921,6 +925,20 @@ def _lower_getattr_tensor_metadta_op(model: QuantizedGraphModule):
             args[0] = n.args[0].args[0]
             n.args = tuple(args)
 
+def _lower_get_tensor_info_op(model: QuantizedGraphModule):
+    """ Modified the graph of the model inplace, to skip extra dequantize op before
+    the general tensor shape ops when possible
+    """
+    for n in model.graph.nodes:
+        if is_get_tensor_info_node(n):
```

Comment:
nit: this can also be:
```
if not is_get_tensor_info_node(n):
    continue
```
to reduce identation

Reply:
OK. It's fixed